arrows/pdal/tests/test_pointcloud_io.cxx

```diff
@@ -39,78 +41,75 @@ class pointcloud_io : public ::testing::Test
 // ----------------------------------------------------------------------------
 TEST_F(pointcloud_io, create)
 {
-  using namespace kwiver::vital;
-  plugin_manager::instance().load_all_plugins();
```

Comment:
There is a reason to load the plugins here an not for all tests.  Other than this special "create" test, it's probably better to directly instantiate the pdal object in the other tests.  This way if there is a failure in the plugin loading it only impacts one test that is directly testing plugin loading and the other tests will still run.  There is no problem having the PDAL tests directly link to the PDAL arrow library.